app/Http/Controllers/SubmissionController.php

```diff
@@ -128,6 +130,35 @@ private function submitProcess(): Response
         } elseif (intval($this->project->Id) < 1) {
             abort(Response::HTTP_NOT_FOUND, 'The requested project does not exist.');
         }
+        if ((bool) config('cdash.validate_xml_submissions') === true) {
```

Comment:
What do you think about running all submissions through the validator?  I think it would be useful to log the result of submission validation for all submissions, and instead use the config setting to control whether mis-formatted submissions are entirely rejected or not.

Reply:
I like that idea.  It seems like it would be useful information about malformed submissions that aren't "rejected"